web3.js/src/transaction.ts

```diff
@@ -542,6 +543,21 @@ export class Transaction {
     this.signatures[index].signature = Buffer.from(signature);
   }
 
+  async getEstimatedFee(connection: Connection): Promise<RpcResponseAndContext<number>> {
+    const feeMessage = this.compileMessage();
```

Comment:
nit: can we just call this `message`? `feeMessage` is a bit confusing.

Reply:
Yeah, that was leftover from a previous implementation I was working on. I meant to clean it up. I'll get this fixed!

Reply:
Since there isn't a whole lot going on in this method, I removed the variable declaration for `this.compileMessage()` and simplified it to one line.

Reply:
Yeah, I like that better, just making it a one-liner.